src/composables/latestTransactionList.ts

```diff
@@ -28,14 +40,39 @@ export function useLatestTransactionList({ store }: IDefaultComposableOptions) {
 
   const tokens = computed(() => store.state.fungibleTokens.tokens);
 
+  const localPendingTransactionsList = computed(
+    (): IPendingTransaction[] => {
+      const { networkId } = store.getters.activeNetwork;
+      return (store.state.transactions as ITransactionsState).pending[networkId] || [];
+    },
+  );
+
+  function buildLatestTransaction() {
```

Comment:
Maybe this function could be replaced with a composable? You already store all the transaction types in refs.

Reply:
I do not see a reason